app/assets/css/main.styl

```diff
@@ -1,100 +1,82 @@
-@require 'normalize'
-@require 'skeleton'
-@require 'grid'
-@require 'colors'
-@require 'typography'
-@require 'lists'
-@require 'header'
-@require 'navmenu'
-@require 'breadcrumbs'
-@require 'cards'
-@require 'search'
-@require 'footer'
-@require 'images'
-@require 'loader'
-@require 'contact'
-
-html
-    height: 100%
+@require 'variables'
 
 body
     margin: 0
-    position: relative
-    min-height: 100%
 
-.main
-    height: 100%
-
-.content
-    position: relative
-    margin-top: 2rem
-
-.drug
-    color: #F06292
-    font-weight: bold
-
-.taxonomy-list
-    a
-        text-decoration: none
-        color: $dark-gray
-        &:hover
-            color: $primary
-
-.cards-list
-    font-size: 2rem
-
-    li
-        list-style: none
-        color: $dark-gray
-        margin: 0 0 0 0
-        padding: 1rem 0
-        border-bottom: 1px solid $light-gray
-        display: block
-        a
-            color: $dark-gray
-            text-decoration: none
-            &:hover
-              color: $primary
-        .metadata
-            display: none
-            font-weight: bold
-            font-size: 1rem
+button
+    cursor: pointer
+    border: 0
+    background: transparent
+    outline: none
 
-.breadcrumbs
-    display: none
+input,
+textarea,
+select
+    height: 38px
+    padding: 6px 10px
+    background-color: white
+    border: 1px solid $border-color
+    border-radius: 2px
+    box-shadow: none
+    box-sizing: border-box
+    &:focus,
+    &:active
+        outline: 0
 
 select
     cursor: pointer
     background: url(/assets/images/down-caret.svg) no-repeat 95% 50%
-    background-size: 1.6rem
+    background-size: 16px
     -webkit-appearance: none
     -moz-appearance: none
     appearance: none
 
-.home h1
-    font-size: 3rem
+input
+    font-size: 20px
+    padding: 6px 10px
+    border-radius: 2px
 
-@media (min-width: 750px)
-    .cards-list
-        li
-            .metadata
-                display: inline
-                margin-left: 2rem
-                a
-                    margin-right: 1rem
-                    background-color: $light-gray
-                    padding: 0.5rem 1rem
-    .backButton
-        display: none
-    .breadcrumbs
-        display: block
+.row
+    width: 100%
+    display: flex
+    margin: $block-margins
+
+.row--stacked
+    flex-direction: column
+
+.row--wrap
+    flex-wrap: wrap
+
+.column
+    padding: 10px
+    flex: auto
+    min-width: 200px
 
-/* Normalize
------------------------------------------*/
+for i in 12 25 50
+    .column--{i}
+        width: ($container-max-width * (i / 100)) - 20
 
-div,
-section,
-article,
-ul,
-table
+.main
     box-sizing: border-box
+    margin: 0
+    min-height: 100vh
+
+.content
+    max-width: $container-max-width
+    margin: auto
+    height: 100%
+    flex: auto
```

Comment:
Making `.content` the only element in this flex group that grows now makes the footer always stick to the bottom.